src/utils/testfunctions.py

```diff
@@ -1,10 +1,66 @@
 import time
+from src.utils.wrappers import time_logger_wrap, exception_wrap
+import time
+import numpy as np
+import pandas as pd
+import logging
+
+LOGGER = logging.getLogger(__name__)
+LOGGER.setLevel(logging.INFO)
+
+@exception_wrap
```

Comment:
This all looks great. No need to add anything else to the functions, because they are just for testing.